Worksheet_D/oxo.py

```diff
@@ -0,0 +1,115 @@
+class OxoBoard:
+    def __init__(self):
+        """ The initialiser. Initialise any fields you need here. """
+        #This command sets up the gameboard as 9 different values each beginning with 0(empty)
+        self.game_board = [0, 0, 0, 0, 0, 0, 0, 0, 0]
+
+    def get_square(self, x, y):
+        """ Return 0, 1 or 2 depending on the contents of the specified square. """
+        #Determines the location of a particular cell on the board using the equation y * 3 + x, so height of the co-ordinate input multiplied by 3 + the width of the co-ordinate input.
```

Comment:
Reasonably good use of commenting. However try to focus on explaining "why" more than "what", and try to avoid simply rephrasing the code in English.

For example, here you might note that the squares are numbered 0 to 8 in order of left-to-right, top-to-bottom, hence where the formula `y * 3 + x` comes from.